### ISSUE TITLE
[Request] Delete mistake sheets created with cutlines 

### ISSUE THREAD

**Issue by rchampine** [ohmg-dev/OldInsuranceMaps#147]
There does not appear to be a way to delete new layers (sheets marked with [x]) that are created from the preparation stage. Since it's possible to accidentally create new sheets from an imprecise cutline, it would be beneficial to be able to remove these extraneous items. Ideally, this ability to delete would only be available on newly created sheets, perhaps only by a subset of users, and not sheets loaded directly from LOC.

Example: [Savannah, Ga. | 1955 | Vol. 1 p50 [3]](https://oldinsurancemaps.net/resource/7825)

<img width="865" alt="image" src="https://github.com/mradamcox/ohmg/assets/136510277/564468db-4aa8-43f9-a989-462c959fcd1b">

**Comment by mradamcox** [comment]
@rchampine There is a way to do this, it is just not well-documented, or very intuitive. You have to go to the resource page for the parent and, if you are logged in, you'll see an "undo" button there in the Preparation section: 
![image](https://github.com/mradamcox/ohmg/assets/10427268/e0d47ba3-540a-49f7-9f08-fb5d15a4c90a). In this particular case, it should work perfectly fine. If one of the "down-stream" documents has already been georeferenced, then the button will be disabled, until the georeferencing session has been deleted which only an admin can currently do.

This whole side of things should really be reworked. I've been thinking for a while about adding a "staff user" level, or something along those lines, which would allow me to add more controls to the frontend that only appear for a few people. Will make a new ticket for that though.

**Comment by rchampine** [comment]
Ah I see! I missed the link to the parent resource page. I just tried this and it seems to have worked — but with one hiccup:

The children of the mistaken cutline are all 404'd now (as expected):
https://oldinsurancemaps.net/resource/7823
https://oldinsurancemaps.net/resource/7824
https://oldinsurancemaps.net/resource/7825

But their previews and dead links still appear under Prepared in the volume summary: https://oldinsurancemaps.net/loc/sanborn01500_008/#prepared

**Comment by mradamcox** [comment]
Ah, ok, that should get cleared up if you click the Repair Summary button on the volume page (at the top of the Georeferencing Overview). If that does work I'll make a separate ticket because that cleanup operation should be triggered by the undo operation.

**Comment by rchampine** [comment]
Nice — the Repair Summary button worked.

**Comment by rchampine** [comment]
I expect having to do this will be pretty rare (and handled by me for DSL students) so I wouldn't prioritize this enhancement. I  think the overall takeaway for me is that a "delete sheet" button on the child page rather than the parent page would be useful/more intuitive (and could preserve any valid crops). Thanks for your help on this!

**Comment by mradamcox** [comment]
Yeah, that makes a lot of sense for sure, it's just that at the moment this split operation is handled as a whole, kind of an all or nothing setup. Some re-modeling that I have in mind could remove that constraint, but for it'll have to stay this way. I could still put an "undo" button on the child sheets with more clear instructions, so I'll leave this ticket open until I can get to that.

**Comment by mradamcox** [comment]
I think the recent improvements to the undo preparation process cover this ticket, closing now.
